### PR TITLE
svg: render spans in different colors and add arguments

### PR DESCRIPTION
**Pull request by zeebo**

### CODE REVIEW

File: present/trace.go [hunk 1]
```diff
@@ -15,8 +15,10 @@
 package present
 
 import (
+	"context"
```
Comment (jtolds) [inline present/trace.go]: nope, now we don't support older go releases.

File: present/trace.go [hunk 2]
```diff
@@ -72,14 +74,24 @@ func SpansToSVG(w io.Writer, spans []*collect.FinishedSpan) error {
 	}
 
 	for id, s := range spans {
+		color := "rgb(128,128,255)"
+		switch {
+		case s.Panicked:
+			color = "rgb(255,0,0)"
+		case unwrapError(s.Err) == context.Canceled:
```
Comment (jtolds) [inline present/trace.go]: i think there's a variable defined for contextCanceled that's go-release-specific somewhere maybe? maybe that's some other thing we've open sourced.

Comment (jtolds) [inline present/trace.go]: oops, that's what we did for spacemonkeygo/errors, not this.

Comment (zeebo) [inline present/trace.go]: aw crap. good catch.

**Pull request merged**